cmd/validators/validators.go

```diff
@@ -49,13 +49,9 @@ func CmdError(err error, commandPath string) error {
 }
 
 func validateOutput(outputFlag string) error {
-	validKinds := map[string]struct{}{
-		"json":      {},
-		"plaintext": {},
-	}
-	_, ok := validKinds[outputFlag]
-	if !ok {
-		return output.ErrInvalidOutputKind
+	_, err := output.NewOutputKind(outputFlag)
```

Comment:
I moved the validation to a constructor function since it's used in a few places.